Makefile

```diff
@@ -24,7 +24,6 @@ ifneq (,$(DAPPER_HOST_ARCH))
 
 PLATFORMS ?= linux/amd64,linux/arm64
 IMAGES = subctl
-PRELOAD_IMAGES := submariner-operator submariner-gateway submariner-globalnet submariner-route-agent lighthouse-agent lighthouse-coredns
```

Comment:
FWICT, the changes in shipyard will now preload $IMAGES which would now only be `subctl`, ie the others specified here won't be preloaded or am I missing something?

Reply:
Yes that's precisely the point, each project tests it's own built images by default, and as a developer you can explicitly preload extra images when you need to (which would be quite rare AFAICT).

Nowadays everything is preloaded from the local docker registry which, most often than not, contains stale images which lead to buggy and unexpected behaviors.